### ISSUE TITLE
line on the right

### ISSUE THREAD

**Issue by ueen** [schlagmichdoch/PairDrop#240]
firefox 121.0
<img width="1086" alt="Bildschirmfoto 2024-01-09 um 10 45 01" src="https://github.com/schlagmichdoch/PairDrop/assets/5067479/c953f938-9b30-4b68-a247-9e44e6ad6118">

**Comment by schlagmichdoch** [comment]
I cannot reproduce this. Are you sure this is not part of the browser?

**Comment by ueen** [comment]
Yes it only appears on pairdrop, it's in Firefox on mac but not Safari.

**Comment by schlagmichdoch** [comment]
Can you select it via dev tools?

**Comment by ueen** [comment]
Good idea, its part of the div with `id=center` but cant select it directly.

**Comment by schlagmichdoch** [comment]
I still cannot reproduce this issue. What version of MacOS are you using?
